### ISSUE TITLE
style is messed up in Rails 5

### ISSUE THREAD

**Issue by merongivian** [denkGroot/Spina#201]
When installing in a new rails project styles are not correct, this is happens to me on rails 5 though not sure if this is the issue:

<img width="1205" alt="spina" src="https://cloud.githubusercontent.com/assets/4277403/20267833/85c70d5a-aa4a-11e6-9921-3cef312dee69.png">

**Comment by Bramjetten** [comment]
Please try using the latest spina-template and spina gems.

**Comment by merongivian** [comment]
@Bramjetten you have any branch, commit in specific?, it seems you are currently using papertrail, which is not working with rails 5

**Comment by Bramjetten** [comment]
The latest commit on the master branch should work.

**Comment by merongivian** [comment]
@Bramjetten this are my conflicts when trying `bundle install`:

```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    spina was resolved to 0.10.0, which depends on
      globalize was resolved to 3.0.0, which depends on
        activerecord (>= 3.0.0)

    spina was resolved to 0.10.0, which depends on
      globalize was resolved to 3.0.0, which depends on
        paper_trail (~> 2) was resolved to 2.6.3, which depends on
          activerecord (~> 3.0)

    rails (~> 5.0.0) was resolved to 5.0.0, which depends on
      activerecord (= 5.0.0)
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (~> 5.0.0)

    spina was resolved to 0.10.0, which depends on
      globalize was resolved to 3.0.0, which depends on
        paper_trail (~> 2) was resolved to 2.0.0, which depends on
          rails (~> 3)

    spina was resolved to 0.10.0, which depends on
```
